### ISSUE TITLE
Right-click > Copy image from browser, then Paste in PicoShare pastes text instead of image

### ISSUE THREAD

**Issue by lsrom** [mtlynch/picoshare#350]
Currently, when I paste image from clipboard, it creates `.txt` file with structure like this:

```
<meta charset='utf-8'><img src="https://some-image-url.jpg"/>
```

Would it be possible to include the image file itself? It works for example in Gmail where you can paste image into the email body.

I'm not a web developer otherwise I would look into this myself, if someone wants to give me some starting point I might try to put this together.

Thanks!

**Comment by mtlynch** [comment]
Thanks for reporting this!

This is currently supported. I've tested it successfully on Chrome and Firefox on Win10.

https://user-images.githubusercontent.com/7783288/202857755-16b5fe0e-26a3-4fdf-b50f-9229908687f2.mp4

Which browser / OS are you seeing this on?

**Comment by lsrom** [comment]
I've just tested it again to make sure. Works if I use Windows snipping tool (Shift + Win + S) or copy image file from Windows file system but doesn't work if I right click image in the browser, select Copy Image and paste it in the "Or paste something here" box. 

Tested the browser behavior in Chrome 107.0.5304.107 on Windows and 107.0.5304.87 on Mac (M1). Copying from the browser works in other applications, tested with web client Gmail and desktop client Viber chat app.
